test/unit/player.test.js

```diff
@@ -478,6 +478,45 @@ test('make sure that controls listeners do not get added too many times', functi
 //   player.dispose();
 // });
 
+test('should select the proper tech based on the the sourceOrder option',
+  function() {
+    var fixture = document.getElementById('qunit-fixture');
+    var html =
+        '<video id="example_1">' +
+          '<source src="fake.foo1" type="video/unsupported-format">' +
+          '<source src="fake.foo2" type="video/foo-format">' +
+        '</video>';
+
+    // Extend TechFaker to create a tech that plays the only mime-type that TechFaker
+    // will not play
+    import Tech from '../../src/js/tech/tech.js';
+    import TechFaker from './tech/tech-faker.js';
```

Comment:
They are required to be at the top of the file. Only reason it hasn't complained is because we're putting babel into `loose` mode.